internal/registry/image.go

```diff
@@ -73,3 +72,32 @@ func ExtractBundleImage(ctx context.Context, logger *log.Entry, image string, lo
 
 	return bundleDir, nil
 }
+
+// GetImageLabels returns the set of labels on image.
+func GetImageLabels(ctx context.Context, logger *log.Entry, image string, local bool) (map[string]string, error) {
```

Comment:
Shows that it is not related to : 

`Changes the scorecard command to look for a namespace in the
      kubeconfig instead of defaulting to 'default' when the 
      namespace is not defined on the command line flag.  If a namespace
      can not be found in the kubeconfig file, then 'default' is used.`

Am i right? Is possible to split the PR and add the context of this other change for we get why and in this way be able to help with the review?